server/engine/src/bad_frames_detector/thresholds.py

```diff
@@ -8,6 +8,20 @@
 
 
 class ThresholdsDetector(AbstractDetector):
```

Comment:
Move this file to `utilities` and remove folder `/bad_frames_detector`? That way, the only other folders are major blocks of the pipeline (or tests).